Gemfile

```diff
@@ -1,11 +1,11 @@
 source 'https://rubygems.org'
 
 ruby '2.3.1'
-gem 'rack', '1.6.5'
-gem 'sinatra'
+gem 'rack', '~> 1'
```

Comment:
This requirement is good as having nothing there really :)

Reply:
sure, but I won't be updating to rack 2 - anything can break!

Reply:
Ah, I now see your other PR - would be good to merge if it works with rack-cors.

Reply:
`~> 1` should map to `~> X` no? It would have to be `< 2`.

Reply:
yep

Reply:
sorry, understood incorrectly: `~> 1` will map to any `1.X.Y` version, below 2.